staging_service/autodetect/Mappings.py

```diff
@@ -29,6 +29,25 @@
 
 MEDIA = "KBaseBiochem.Media"
 
+fastq_reads_id = "fastq_reads"
+sra_reads_id = "sra_reads"
+genbank_genome_id = "genbank_genome"
+gff_genome_id = "gff_genome"
+gff_metagenome_id = "gff_metagenome"
+expression_matrix_id = "expression_matrix"
+media_id = "media"
+fba_model_id = "fba_model"
+assembly_id = "assembly"
+phenotype_set_id = "phenotype_set"
+sample_set_id = "sample_set"
+
+# To be added
```

Comment:
Is this comment still valid? At least some of these are in the json

Reply:
These are to be added to https://github.com/kbase/narrative/blob/develop/kbase-extension/static/kbase/config/staging_upload.json, updated the comment